lib/Doctrine/DBAL/Migrations/MigrationException.php

```diff
@@ -1,63 +1,97 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Doctrine\DBAL\Migrations;
 
-use \Doctrine\DBAL\Migrations\Finder\MigrationFinderInterface;
+use Doctrine\DBAL\Migrations\Finder\MigrationFinderInterface;
+use function get_class;
+use function sprintf;
 
-/**
- * Class for Migrations specific exceptions
- *
- * @license     http://www.opensource.org/licenses/lgpl-license.php LGPL
- * @link        www.doctrine-project.org
- * @since       2.0
- * @author      Jonathan H. Wage <jonwage@gmail.com>
- */
 class MigrationException extends \Exception
```

Comment:
This thing should be exterminated for 2.0 in a similar way to https://github.com/doctrine/doctrine2/pull/6743 and https://github.com/doctrine/doctrine2/pull/7210.